WebHostLib/stats.py

```diff
@@ -65,12 +112,15 @@ def stats():
         current_angle += angle
         data["end_angles"].append(current_angle)
 
-    data["colors"] = [element[1] for element in sorted((game, color) for game, color in
-                                                       zip(data["games"], cycle(palette)))]
-    pie.wedge(x=0.5, y=0.5, radius=0.5,
+    data["colors"] = [game_to_color[game] for game in data["games"]]
+
+    pie.wedge(x=0, y=0, radius=0.5,
               start_angle="start_angles", end_angle="end_angles", fill_color="colors",
               source=ColumnDataSource(data=data), legend_field="games")
 
-    script, charts = components((plot, pie))
+    per_game_charts = [create_game_played_figure(games_played, game, game_to_color[game]) for game in total_games
+                       if total_games[game] > 1]
```

Comment:
any particular reason for the `>1` ? This made my testing awkward

Reply:
A graph with a single datapoint just becomes a big rectangle in bokeh vbars